### PR TITLE
feat: refresh float

### LINKED ISSUE #228: feat: refresh describe resource on change
I usually want to follow when I describe an object,
can the float refresh the output?
Thanks.

### PR DESCRIPTION
**Pull request by Ramilito**
Fixes #228 

This is working as we want it I believe, do you want to verify @mosheavni?

- [x] Don't clear content if we only want to refresh the same float view
- [x] Don't clear builder state if it's a refresh of the float_view
- [x] handle informer events (from events resource) not triggering eventhandler for other resources
- [ ] fix ugly hacks (read comments)

### CODE REVIEW

File: lua/kubectl/mappings.lua [hunk 1]
```diff
@@ -161,6 +161,14 @@ function M.register()
           view = require("kubectl.views.fallback")
           view.View()
         end
+      else
+        local _, buf_name = pcall(vim.api.nvim_buf_get_var, 0, "buf_name")
+        local parsed_buf_name = string.lower(string_utils.trim(string.match(buf_name, "([^_]+)")))
+        local ok, view = pcall(require, "kubectl.views." .. parsed_buf_name)
+
+        if ok then
+          pcall(view.Desc)
```
Comment (Ramilito) [inline lua/kubectl/mappings.lua:170]: We should figure this out from the second part of the buf_name

**Comment by mosheavni** [comment]
found a little issue: when i first describe a pod, then quit the float, then describe a different pod, I see the last output. I have to close it again and re-describe it in order to see the right pod description.

**Comment by mosheavni** [comment]
> found a little issue: when i first describe a pod, then quit the float, then describe a different pod, I see the last output. I have to close it again and re-describe it in order to see the right pod description.

oh i was wrong, it was just loading but now it didn't display the "Loading..." when it actually did make sense to display it for every new describe

**Comment by Ramilito** [comment]
you scared me there with the first comment 😅! I'll do some small beauty fixes to the code and then merge it

**Comment by Ramilito** [comment]
@mosheavni The issue you mentioned should be fixed now, if we find any more lets open new pr's since this one is getting too big now

**Pull request merged**